src/Analyzer.Utilities/Core/Extensions/StringExtensions.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
I am assuming this is just moving the source file under the Core folder, correct?